pkg/esp8266_sdk/Makefile

```diff
@@ -0,0 +1,40 @@
+PKG_NAME=esp8266_sdk
+PKG_URL=https://github.com/espressif/ESP8266_RTOS_SDK
+# This is a version in the v3.1 release branch, between 3.1 and 3.1.1.
+PKG_VERSION=913a06a9ac3b2f18009e8fee8f092ca9ffeccd38
+PKG_LICENSE=Apache-2.0
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+ifneq ($(CPU),esp8266)
+  $(error This package can only be used with the ESP8266 CPU)
+endif
```

Comment:
Instead of this, you could add a Makefile.dep file to the package with the following content:

```mk
# This package can only be used with the ESP8266 CPU
FEATURES_REQUIRED += arch_esp8266
```

Reply:
Done.